projects/models.py

```diff
@@ -8,7 +8,22 @@ class ModelBase(models.Model):
     class Meta:
         abstract = True
 
+class Client(models.Model):
+    department = models.CharField(max_length=255, blank=True)
+    agency = models.CharField(max_length=255, blank=True)
+    omb_agency_code = models.CharField(max_length=255, blank=True)
+    omb_bureau_code = models.CharField(max_length=255, blank=True)
+    treasury_agency_code = models.CharField(max_length=255, blank=True)
+    cgac_agency_code = models.CharField(max_length=255, blank=True)
+
+    def __str__(self):
+        return '%s - %s' % (self.department, self.agency)
+
+    class Meta:
+        verbose_name = 'Client'
```

Comment:
sorry, lines 22-24 -- you're overriding the default verbose_name (https://docs.djangoproject.com/en/1.9/ref/models/options/#verbose-name), not sure we need to